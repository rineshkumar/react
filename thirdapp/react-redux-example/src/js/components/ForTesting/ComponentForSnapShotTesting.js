const ComponentForSnapShotTesting = () => {
    return (
        <div>
            <h1>hello World 123</h1>
        </div>
    )
}

export default ComponentForSnapShotTesting