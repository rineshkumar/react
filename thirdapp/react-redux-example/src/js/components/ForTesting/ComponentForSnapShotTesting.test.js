
import React from "react"
import ComponentForSnapShotTesting from "./ComponentForSnapShotTesting"
import '@testing-library/jest-dom';
import Enzyme, { shallow } from "enzyme"
import toJson from "enzyme-to-json"
import Adapter from "@wojtekmaj/enzyme-adapter-react-17"
Enzyme.configure({ adapter: new Adapter() })
it("Testing the basic component", () => {
    const wrapper = shallow(<ComponentForSnapShotTesting />)
    expect(toJson(wrapper)).toMatchSnapshot()
})