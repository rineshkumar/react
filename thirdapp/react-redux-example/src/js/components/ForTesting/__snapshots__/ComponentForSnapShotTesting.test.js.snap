// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the basic component 1`] = `
<div>
  <h1>
    hello World 123
  </h1>
</div>
`;
